certified-connectors/Morf/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0c425a",
+    "capabilities": [],
+    "publisher": "AFTIA Solutions"

```

Comment:
@GuillaumeCleme Can you please add the `"stackOwner"` property as well?

Docs - [Add Metadata](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3a-publisher-and-stackowner-properties)

Reply:
Property added. @sriyen-msft